stingray/lightcurve.py

```diff
@@ -1004,13 +1104,16 @@ def sort(self, reverse=False):
             The :class:`Lightcurve` object with sorted time and counts
             arrays.
         """
-
         new_time, new_counts, new_counts_err = \
             zip(*sorted(zip(self.time, self.counts, self.counts_err),
                         reverse=reverse))
+        new_time = np.asarray(new_time)
+        new_counts = np.asarray(new_counts)
+        new_counts_err = np.asarray(new_counts_err)
```

Comment:
Why do we need to impose the `asarray` here? Won't that be done in `Lightcurve.__init__` anyway?

Reply:
It does nothing if it's already an array, seemed like a harmless addition